docs/user-manual/modules/ROOT/pages/xml-configuration.adoc

```diff
@@ -1,25 +1,25 @@
 [[XMLConfiguration-XMLConfiguration]]
 === XML Configuration
 
-We recommend developers use the Java link:dsl.adoc[Domain Specific
+We recommend developers use the Java xref:dsl.adoc[Domain Specific
 Language] when writing routing rules as this provides maximum IDE
 completion and functionality while being the most expressive. However if
 you wish to put your routing rules in XML you can via the Camel XML
 language.
 
-Camel XML uses link:spring.adoc[Spring] 2 namespaces; so that you can
-configure your routing rules within your link:spring.adoc[Spring XML
+Camel XML uses xref:spring.adoc[Spring] 2 namespaces; so that you can
+configure your routing rules within your xref:spring.adoc[Spring XML
 configuration file]; you can also use
-link:faq/how-do-i-configure-endpoints.adoc[Java code to configure components
+xref:ROOT:faq/how-do-i-configure-endpoints.adoc[Java code to configure components
```

Comment:
I think the issue here is that the [FAQ](https://github.com/apache/camel/tree/master/docs/user-manual/en/faq) is not in the correct directory according to Antora's [module tree structure](https://docs.antora.org/antora/2.0/modules/).

I think we need to move the content of the [faq](https://github.com/apache/camel/tree/master/docs/user-manual/en/faq) to [ROOT/pages](https://github.com/apache/camel/tree/master/docs/user-manual/modules/ROOT/pages) and create entries for those files in the `user-manual-nav.adoc.template` file under a `Frequently asked questions` heading, similar to how we have it for the [Domain Specific Languages](https://github.com/apache/camel/blob/master/docs/user-manual-nav.adoc.template#L38-L42)

Then, this would simply be:

```suggestion
xref:how-do-i-configure-endpoints.adoc[Java code to configure components
```